modules/ROOT/pages/deployment/wopi/wopi.adoc

```diff
@@ -0,0 +1,103 @@
+= Office Applications Using WOPI
+:toc: right
+:wopi_subdir: /ocis_wopi
+:download-gh-directory-url: https://download-directory.github.io
+:traefik_url: https://traefik.io/traefik/
+:nginx_url: https://www.nginx.com
+:apache_url: https://www.apache.org
+
+:description: Infinite Scale uses the WOPI protocol to integrate office applications. This document describes the general flow and basic settings used.
+
+== Introduction
+
+{description} For more details on possible configuration values see the xref:conf-examples/office-integration.adoc[Office Integration] document at the configuration examples. 
```

Comment:
Would it make sense to add a reference to https://en.wikipedia.org/wiki/Web_Application_Open_Platform_Interface or even https://docs.microsoft.com/en-us/openspecs/office_protocols/ms-wopi/6a8bb410-68ad-47e4-9dc3-6cf29c6b046b here to add some background about what WOPI actually is?

Reply:
added